pkg/skaffold/server/server.go

```diff
@@ -174,9 +193,18 @@ func newHTTPServer(port, proxyPort int) (func() error, error) {
 	}
 	logrus.Infof("starting gRPC HTTP server on port %d", port)
 
+	server := &http.Server{
+		Handler:     mux,
+		ReadTimeout: 10 * time.Second,
```

Comment:
why is this required?

Reply:
removed the `ReadTimeout` since its not required. 
Fixed the `go http.serve(l, mux) to `go server.Serve()`